src/TMxpEntityTagHandler.cpp

```diff
@@ -52,7 +53,17 @@ TMxpTagHandlerResult TMxpEntityTagHandler::handleStartTag(TMxpContext& ctx, TMxp
         if (tag->hasAttribute("PUBLISH")) {
             client.publishEntity(entity, value);
         }
-    }
+    } else if (!tag->hasAttribute("ADD") && !tag->hasAttribute("REMOVE")) {
+        // Apparently there is no (or an empty) value, so set us to empty string.
+        // Adding or removing an empty string to the value has no effect, that's why we skipped those.
+        // Note that the TagParser will parse a '' argument as a non existent argument, so
+        // so both <!EN name> and <!EN name ''> will come here.
+        const QString& value = "";
+        resolver.registerEntity(entity, value);
```

Comment:
Actually there is no need to explicitly declare a variable - just use the default constructor directly. The null `QString` it creates is almost completely indistinguishable from an *empty* `QString` - in fact the only thing that can tell the difference between them is the `QString::isNull()`  method! :astonished: 
```suggestion
        resolver.registerEntity(entity, QString());
```

:information_source:  \<aside>*Also we try to avoid the use of raw C string literals to initialise **non-empty** `QString`s - because other coders cannot tell whether the text is intended to be put through the Qt translation system (and fed to the *CrowdIn* translation team) or to be left out of that (typically when the text is used within the Lua sub-system which uses explicitly American English language). As such the former type of strings will have a [`QObject::tr(`...`)`](https://doc.qt.io/qt-5/qobject.html#tr) or [`QTranslator::translate(`...`)`](https://doc.qt.io/qt-5/qtranslator.html#translate) wrapper and the latter a [`QStringLiteral(`...`)`](https://doc.qt.io/qt-5/qstring.html#QStringLiteral) - which we have shortened to `qsl(`...`)` or a [`QLatin1String(`...`)`](https://doc.qt.io/qt-5/qlatin1string.html) wrapper around it.*\</aside>